common.h

```diff
@@ -257,7 +257,7 @@ namespace bvolderivatives {
  * back and forth.
  */
 namespace fsgrids {
-   enum bfield {
+   enum bfield : int {
```

Comment:
For my own edification, what does this `int?` do? Aren't enums mapping to ints anyway?

Reply:
They *should* map to the smallest integer type which can contain the whole enum. However, with this templating business, I was getting issues with the correct template type not being found, so had to declare this explicitly to ensure compilation. A bit weird, but no harm.

Reply:
Probably, but it turns out that's impementation defined.

Reply:
OK so template+compiler taming, fine by me.